packages/accordion/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "mui-accordion",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/accordion/src",
+  "projectType": "library",
+  "tags": [],
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "options": {
+        "eslintConfig": ".eslintrc.json",
+        "lintFilePatterns": ["packages/accordion/**/*.{js,ts}"],
+        "silent": false,
+        "fix": false,
+        "cache": true,
+        "cacheLocation": "./node_modules/.cache/accordion/.eslintcache",
+        "maxWarnings": -1,
```

Comment:
is there a reason maxWarnings is -1 instead of 0? idk if thats just an nx convention i'm just unfamiliar with but it looks kinda funny